package.json

```diff
@@ -41,7 +41,11 @@
     "react-router-dom": "^6.3.0",
     "react-scripts": "^5.0.1",
     "typescript": "^4.7.2",
-    "web-vitals": "^2.1.4"
+    "web-vitals": "^2.1.4",
+    "@emotion/react": "^11.9.3",
+    "@table-library/react-table-library": "^4.0.10",
+    "install": "^0.13.0",
+    "npm": "^8.15.0"
```

Comment:
This might have been a mistake. Is it really necessary?

Reply:
Yep I'm just cleaning up these things. Code is not quite ready for review. 😉

Reply:
Ah, okay sorry! Take your time. : D